tools/src/launcher/Main.hs

```diff
@@ -15,26 +15,27 @@ import           Universum
 
 import           Control.Concurrent (modifyMVar_)
 import           Control.Concurrent.Async.Lifted.Safe (Async, async, cancel, poll, wait, waitAny,
-                                                       withAsyncWithUnmask)
+                                                       withAsync, withAsyncWithUnmask)
 import           Control.Exception.Safe (tryAny)
 import           Control.Lens (makeLensesWith)
 import qualified Data.ByteString.Lazy as BS.L
 import           Data.List (isSuffixOf)
+import           Data.Maybe (isNothing)
 import qualified Data.Text.IO as T
 import           Data.Time.Units (Second, convertUnit)
+import           Data.Tuple.Select (sel3)
```

Comment:
It's not written anywhere, but we don't use `tuple` library. We use functionality from `lens` instead. `sel3` is `^. _3`. Both `^.` and `_3` are exported from `Universum`.